app/assets/stylesheets/main.scss

```diff
@@ -11,7 +11,7 @@ $navbar-pf-alt-navbar-toggle-icon-bar-hover-bg: #fff;
 $img-bg-navbar:                             "layout/navbar.png"; // sets a custom background image in the header
 $color-success:                             #3f9c35;
 $color-warning:                             #ec7a08;
-$color-critical:                            #cc0000;
+$color-critical:                            #c00;
```

Comment:
Same here, just because "#cc0000" can also be written as `#c00` doesn't mean we should, especially in a list where all the other colors use the long name.